client/my-sites/domains/domain-management/edit/domain-types/style.scss

```diff
@@ -0,0 +1,7 @@
+.card {
+	.gridicons-trash {
+		margin-top: -2px;
+		margin-right: 4px;
+		vertical-align: middle;
+	}
```

Comment:
Do we really need to duplicate [all those rules](https://github.com/Automattic/wp-calypso/blob/d5a5d38bac742109911789babad6af99611f572e/client/me/purchases/remove-purchase/style.scss#L12)? It seems they are already available from `domain.css`:

![screenshot](https://user-images.githubusercontent.com/594356/77063668-5f416980-69de-11ea-9502-72b146214482.png)

Reply:
It's not ideal but the duplicated style is needed for this:

```
<CompactCard href={ link }>
    <Gridicon icon="trash" />
    { title }
</CompactCard>
```